### DISCUSSION TITLE
Shelly 1mini Gen3 support

### DISCUSSION THREAD

**Discussion by dippes** [arendst/Tasmota#21624] [Support request] [answered]
Hello is the Shelly 1 Mini Gen 3 supported?

Is this the same firmware as the Plus1Mini ?

**Comment by sfromis** [comment, marked as answer]
Based on the ESPHome config for the gen 3, it looks like it is using ESP32-C3 instead, meaning that `tasmota32c3.factory.bin` should be usable, if configuring same gpios in the template.

**Comment by Jason2866** [comment]
OTA convert should work too https://github.com/tasmota/mgos32-to-tasmota32

**Comment by dippes** [comment]
Ota Convert worked. The latest firmware was also installed.
Since I prefer the DE version, I tried to upgrade and

entered this URL "http://ota.tasmota.com/tasmota32/release/tasmota32-DE.bin" and started the upgrade.

Unfortunately, Shelly only starts in SafeBoot.

How can I fix the problem?

**Reply by barbudor** [reply]
If it is an ESP32 C3, loading a firmware dedicated to ESP32 is likely to fail

**Comment by dippes** [comment]
I have now updated the firmware with this "http://ota.tasmota.com/tasmota32/release/tasmota32c3.bin". The Shelly starts normally again.

How do I get a German WIF?

**Reply by arendst** [reply]
You'll have to compilw yourselves; we do not provide translated c3 binaries.

**Reply by drbeka** [reply]
> I have now updated the firmware with this "http://ota.tasmota.com/tasmota32/release/tasmota32c3.bin". The Shelly starts normally again.
> 
Hi, 
have you used the original shelly web UI and dropped that tasmota32c3.bin,
or have you followed the workflow in https://github.com/tasmota/mgos32-to-tasmota32 with that multiple steps?

What is difference between that binary and the zip archive https://github.com/tasmota/mgos32-to-tasmota32/releases/download/v14.1.1/mgos32-to-tasmota32-MiniPMG3.zip?
I am asking because there is a big warning to "convert to tasmota safe boot" to unlock the boot loader

**Reply by sfromis** [reply]
The only OTA way of converting to Tasmota is the mgos32 route. The warnings you refer to are important, in addition to Matter being a showstopper, recent Shelly firmware versions became "less open" to the techniques used for OTA firmware replacement.

Alternatively you can use wired flashing, as long as your device is not locked due to Matter support. The firmware updating link is when already having Tasmota, and upgrading to another build. For wired flashing, you'd need a binary like `tasmota32c3.factory.bin`
